src/blend/src/Shared/Blend/Blend.lua

```diff
@@ -548,6 +575,48 @@ function Blend.Children(parent, value)
 	end
 end
 
+--[=[
+	Mounts attributes to the parent and returns an object which will cleanup and delete
+	all attributes when removed.
+
+	```lua
+	local attribute = Blend.State("abc")
+	Blend.New "ScreenGui" {
+		Parent = game.Players.LocalPlayer.PlayerGui;
+		[Blend.Attributes] = {
+			Attribute1 = attribute;
```

Comment:
Unfortunately, ComputedPairs does not let us take in an [Observable] = [Observable] yet, which would make sense in certain situations.

I think this can be a future problem.

Reply:
In that case I've added a bit of extra documentation here to better convey that + an optional workaround for now.